user-service/src/main/resources/application-dev.yaml

```
@@ -0,0 +1,34 @@
+  client-secret: ${KEYCLOAK_CLIENT_SECRET:}
+  admin-username: ${KEYCLOAK_ADMIN_USERNAME:admin}
+  admin-password: ${KEYCLOAK_ADMIN_PASSWORD:admin}
+
+management:
+  tracing:
+    sampling:
+      probability: 1.0
+  zipkin:
+    tracing:
+      endpoint: ${ZIPKIN_ENDPOINT:http://zipkin:9411/api/v2/spans}
+
```

YAMLlint (1.38.0)

[error] too many blank lines (1 > 0)

(empty-lines)